posting/account.py

```
@@ -0,0 +1,160 @@
+
+        pks = [int(x) for x in ids.split(',') if x.strip()]
+
+        accounts = Account.objects.filter(pk__in=pks).select_related()  # оптимизируем
+
+        # Сформируем список форм: для каждого account — PlannedPostForm
+        form_objects = []
+        if request.method == 'POST':
+            # POST: создаём формы из incoming data
+            valid = True
+            created = []
+            for i, acc in enumerate(accounts):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)